src/components/Checkbox/Checkbox_deprecated.js

```diff
@@ -101,6 +98,7 @@ function Checkbox(props) {
       <InternalCheckbox
         name={name}
         inputId={inputId}
+        // eslint-disable-next-line react/prop-types
```

Comment:
why need this ?

Reply:
because I removed the proptypes due to them conflicting with the type definitions, but then it complains that it needs proptypes.